robotiq_hardware_tests/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>robotiq_hardware_tests</name>
+  <version>0.0.1</version>
+  <description>ROS2 driver for the Robotiq gripper.</description>
```

Comment:
What does this package actually do? who is the intended user? should it be `robotiq_hardware_utils` is it simple commands for checking that hardware is functioning as expected? or is it tests that don't need to be distributed via a ros package but could just be internal to the robotiq_driver package for development but not packages and installed for end users?

Reply:
They are commands (to be run in the terminal) to check that the hardware is properly connected and working as expected.
If you think "robotiq_hardware_utils" is more appropriate I do not mind renaming it.